app/assets/stylesheets/common/base/colorpicker.scss

```diff
@@ -35,16 +35,10 @@
       flex: 0 0 auto;
       &.used-color:before {
         position: absolute;
-        top: 0;
-        left: 0;
-        display: flex;
-        align-items: center;
-        justify-content: center;
-        font-family: fontawesome;
-        content: "\f00c";
-        height: 100%;
-        width: calc(1.25em - 2px);
-        color: rgba($secondary, 0.75);
+        top: 0px;
+        left: 2px;
+        content: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="11px" height="11px" viewBox="0 0 512 512" fill="white"><path d="M173.898 439.404l-166.4-166.4c-9.997-9.997-9.997-26.206 0-36.204l36.203-36.204c9.997-9.998 26.207-9.998 36.204 0L192 312.69 432.095 72.596c9.997-9.997 26.207-9.997 36.204 0l36.203 36.204c9.997 9.997 9.997 26.206 0 36.204l-294.4 294.401c-9.998 9.997-26.207 9.997-36.204-.001z" /></svg>');
```

Comment:
Here (and in a few other similar cases) it was complicated to replace a font icon used in a pseudoselector with an icon in the DOM, so I opted to use SVG in CSS directly. It's not ideal, but it helps keep down the size of the PR.

Reply:
I don't love this. I think I'd prefer to remove the icon altogether if we can't easily put it in the DOM. It's for the color picker? It doesn't have a spot for HTML?